src/storm/modelchecker/blackbox/bMDP.h

```diff
@@ -1,25 +1,90 @@
-//
-// Created by Maximilian Kamps on 25.05.23.
-//
-
 #ifndef STORM_BMDP_H
 #define STORM_BMDP_H
 
-class bMDP {
-    //sparse matrix
-    //ValueType: in dieser Datei(nicht notwendigerweise im Header!
-        // 2 doubles
-        // + , constructor for 0
+#include "models/sparse/NondeterministicModel.h"
+#include "storm/adapters/RationalFunctionAdapter.h"
+#include "storm/exceptions/InvalidArgumentException.h"
+#include "storm/models/sparse/StandardRewardModel.h"
+#include "storm/models/sparse/StateLabeling.h"
+#include "storm/utility/constants.h"
+#include "storm/utility/vector.h"
+
+
+namespace storm {
+namespace models {
+namespace sparse {
+
+/*!
+* Template class to store the bounds of the bMDP
+* ValueType of bMDP
+*/
+template<typename ValueType>
+class ValueTypePair {
```

Comment:
Vielleicht sollte das in eine Extradatei und für die INTERVAL-Function als Rückgabetyp benutztbar sein.

Reply:
Ich hab probiert ValuetypePair rauzuziehen, hab dann aber nur kompilierfehler bekommen. Wäre nett, wenn jemand sich das anschauen könnte. :)